tests/legacy-cli/e2e/tests/misc/support-safari-10.1.ts

```diff
@@ -43,16 +43,16 @@ export default async function () {
     `);
   } else {
     await expectFileToMatch('dist/test-project/index.html', oneLineTrim`
+      <script src="runtime-es2015.js" type="module"></script>
```

Comment:
This doesn't seem right to be included prior to the `nomodule` polyfill, as this will cause both runtime to be executed on older versions on Safari 10.1

Reply:
I don't know why `test-browsers` is not failing :(

Reply:
Good call. Done.

Safari 10.1 can parse and execute both runtime chunks so the second one replaces the first.  This would fail if SRI was enabled.